docs/rules/valid-jsdoc.md

```diff
@@ -26,6 +26,7 @@ This rule aims to prevent invalid and incomplete JSDoc comments. It will warn wh
 * A comment for a function is missing `@returns`
 * A parameter has no associated `@param` in the JSDoc comment
 * `@param`s are out of order with named arguments
+* `@return` is used on a function with no `return` keyword without `@abstract` or `@virtual`
```

Comment:
I think we can just remove this. This lost already doesn't mention everything the rule does, and this bullet is a bit hard to understand.